package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postinstall": "bash pomander.sh",
+    "eslint": "eslint",
+    "lint": "eslint ."
```

Comment:
Per our conversation and posterity, eslint and lint are here for running lint manually. Postinstall will run a shell script that will automatically run the linter on every commit.